.travis.yml

```diff
@@ -24,8 +24,7 @@ before_install:
   - sudo apt-get install -y conntrack
   - make setup/travis
 
-# Since there's no persistence sharing, we need to run everything in a single stage
-install: make $TESTS
+install: make ${TESTS} NAMESPACE=default
```

Comment:
At this point, I'm almost certain our goveralls need to be rebuild. I'm not sure how, but there must be an option to collect multiple coverage reports in their service.